Add how-to page to override namespace quota (#68)
appuio/appuio-io-docs: docs/modules/ROOT/pages/how-to/day2ops/override-namespace-quota.adoc

= Override Namespace Quota

This guide describes how a cluster admin can override the namespace/project quota for an organization in a single {zone}.

== Prerequisites

* Administrator access to the {zone}

== Steps

=== Method 1: Create ConfigMap manually

> The override is active as soon as you create a config map for the organization, but only for the next namespaces to be created.

Reviewer: ```suggestion
The override is active as soon as you create a `ConfigMap` for the organization.
```

I'm not sure the second part of the sentence is required at all, it should be obvious to a cluster admin that such a change won't retroactively allow namespaces to have been created in the past (unless I'm misunderstanding what you wanted to say with the second part).

[source,yaml]
----
kind: ConfigMap
apiVersion: v1
metadata:
  namespace: appuio-cloud <1>
  name: override-example-company <2>
data:
  namespaceQuota: '50' <3>
----
<1> The namespace in which the component is installed.
<2> The name follows the pattern `override-?` where `?` is the organization name as defined in `appuio.io/organization` annotation.
<3> Set the new quota (as a string).

> TIP: `oc -n <namespace> create cm override-<org> --from-literal=namespaceQuota=10`

Reviewer: ```suggestion
TIP: You can use `oc -n <namespace> create cm override-<org> --from-literal=namespaceQuota=10` to create an appropriate `ConfigMap` with a single command.
```

> === Method 2: Configure in cluster catalog

Reviewer: It would be nice to explicitly document what happens when I configure a ConfigMap via the cluster catalog which already exists on the system. I assume that ArgoCD will just adopt and update the existing ConfigMap in that case.

Also, I think we should promote doing such configurations via the cluster catalog as the first method.

Author: First part: I agree.
Second: In https://github.com/appuio/component-appuio-cloud/pull/38 we discussed that neither approach is actually preferred. Configuring it via Commodore is a method that should be replaced later with a better APPUiO-native mechanism

= Override Namespace Quota

This guide describes how a cluster admin can override the namespace/project quota for an organization in a single {zone}.

== Prerequisites

* Administrator access to the {zone}

== Steps

=== Method 1: Configure in cluster catalog

> This method uses the component parameters to generate and maintain the `ConfigMaps` for the overrides.

Reviewer 2: Maybe link to the component (docs) for reference?
